src/SDK/Client/Platform/methods/contracts/get.spec.ts

```diff
@@ -14,17 +14,20 @@ const factory = new DataContractFactory(
 const dpp = {
     dataContract: factory
 }
+const getDataContract = async (id) => {
+    switch (id) {
+        // @ts-ignore
+        case contractsFixtures.ratePlatform.id:
+            const contract = await dpp.dataContract.createFromObject(contractsFixtures.ratePlatform);
+            return contract.serialize()
+        default:
+            return null;
+    }
+};
 const client = {
-    getDataContract: async (id) => {
-        switch (id) {
-            // @ts-ignore
-            case contractsFixtures.ratePlatform.id:
-                const contract = await dpp.dataContract.createFromObject(contractsFixtures.ratePlatform);
-                return contract.serialize()
-            default:
-                return null;
-        }
-    },
+    getDAPIClient: () => {
+        return { client: { getDataContract } };
```

Comment:
Just `{  getDataContract  };` ?

Reply:
Yeah, no need to mock more on this one, only getDataContract is needed to get a contract =D